package.json

```diff
@@ -32,6 +32,8 @@
     ]
   },
   "devDependencies": {
+    "@types/history": "^4.7.3",
+    "@types/jest": "^24.0.21",
```

Comment:
Are these dependencies needed?

Reply:
I think those were added by Sethi to help us work out what we needed to do for testing

Reply:
okay just never seen them before! thanks :)